### PR TITLE
added walder,werber map data

### PR DESCRIPTION
**Pull request by DonLoron**

### CODE REVIEW

**Comment by schuer** [comment]
Hey, the map is for people only, not companies. Glad you 3 want to join, you’re very much welcome, but can you add an entry for every single person please, thx!

**Comment by schuer** [comment]
Hint: adjust your markers a little so that they don’t stack on top of each other ;)

File: _directory/data/walderwerber/data.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+# bio
+# describe yourself, your involvement in the project and the community or whatever is relevant for you.
+# should be 300 chars at maximum. no markup or line breaks!
+bio: "Hey guys! We're a team of 3 working at walder,werber, a small marketing agency located at uster, switzerland."
+
+# image or gravatar
+# provide image file from current folder (like 'johndoe.jpg') or gravatar url (generated via https://3v4l.org/OXG7H)
+# should be square and about 200-300 px size!
+#image: johndoe.jpg
```
Comment (staabm) [inline _directory/data/walderwerber/data.yml]: I guess this needs to be left blank instead of commented out.

Comment (schuer) [reply _directory/data/walderwerber/data.yml]: Technically, it both works.

**Comment by schuer** [comment]
🎉

**Pull request merged**